### PR TITLE
Eslint & Prettier intellisense + Package updates & changes

### PR DESCRIPTION
**Pull request by sushanyadav**
## Description
- Eslint + Prettier semi intellisense
- Updates all the packages to latest including NEXT.js which is now 14.0.1
- made changes according to updates

### Screenshots / Videos
- N/A

### How to test?
CleanShot 2023-11-01 at 9  46 02

CleanShot 2023-11-01 at 10  53 05

### CODE REVIEW

**Review by ayungavis — APPROVED** [review]
LGTM, thanks for upgrading the dependencies 🙌

**Comment by sushanyadav** [comment]
Thank you @ayungavis for the review. 

Btw I also, added auto import when on save. wdyt?
cc @Aztriltus

File: .eslintrc.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
+ * @type {import("eslint").Linter.Config}
+ */
 module.exports = {
-  extends: ["plugin:react/recommended", "next/core-web-vitals"],
+  extends: ['plugin:react/recommended', 'next/core-web-vitals'],
   env: { es6: true },
   rules: {
-    "@next/next/no-img-element": "off",
-    "react/jsx-sort-props": [
+    '@next/next/no-img-element': 'off',
```
Comment (Aztriltus) [inline .eslintrc.js:8]: HAHA thanks for this

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [16.x]
+        node-version: [18.x]
```
Comment (Aztriltus) [inline .github/workflows/main.yml:11]: Nice

**Review by Aztriltus — APPROVED** [review]
LGTM, thanks for the updates @sushanyadav!

**Pull request merged**